src/Raven.Client/ServerWide/DatabaseTopology.cs

```diff
@@ -152,6 +152,8 @@ public void Update(DatabaseTopology topology)
             Stamp = topology.Stamp;
             PriorityOrder = topology.PriorityOrder;
             NodesModifiedAt = topology.NodesModifiedAt;
+
+            ReplicationFactor = topology.ReplicationFactor;
```

Comment:
`DynamicNodeDistribution` is also not updated here. Is it on purpose?

Reply:
I guess not :)

Reply:
Let's we fix this (+test) here as well

Reply:
I don't think there is any path that updates the distribution through this command. The dynamic-node-distribution toggle endpoint is not for sharding and also doesn't use `UpdateTopologyCommand`